main.go

```diff
@@ -279,25 +286,11 @@ func newLogger(format, level string, pretty bool) *logrus.Entry {
 	return logger.WithField("type", logConf.TypeFieldKey).WithFields(fields)
 }
 
-func getWatchFilesList(filename, dirPath string) (map[string]struct{}, error) {
+func getWatchFilesList(watchFiles []string) (map[string]struct{}, error) {
```

Comment:
I think, we can have a problem here if a new configuration file is added to a directory or removed from them...

Reply:
Not sure if this is how `watch -d …` is supposed to work. But if we want that, we'll have to keep track of the directory names…